AEPAnalytics/Sources/Analytics.swift

```diff
@@ -276,14 +276,17 @@ public class Analytics: NSObject, Extension {
     /// `EventType.analytics` and `EventSource.requestIdentity`
     /// - Parameter event: The `Event` to be processed.
     private func handleAnalyticsRequestIdentityEvent(_ event: Event) {
-        if let eventData = event.data, !eventData.isEmpty {
-            if let vid = eventData[AnalyticsConstants.EventDataKeys.VISITOR_IDENTIFIER] as? String, !vid.isEmpty {
-                // set VID request
-                updateVisitorIdentifier(event: event, vid: vid)
-            }
-        } else { // get AID/VID request
-            retrieveAnalyticsId(event: event)
+        if analyticsState.privacyStatus == .optedOut {
+            Log.debug(label: LOG_TAG, "handleAnalyticsRequestIdentityEvent - Privacy is opted out, ignoring the update visitor identifier request.")
+            return
```

Comment:
It looks like this method handles two cases; 1) set VID and 2) get VID/AID. With this change, if get AID/VID is called and privacy is opted out, then no response is returned. Can you change this method to perform this check only when setting the VID?